src/libs/TransactionUtils.ts

```diff
@@ -83,14 +83,19 @@ function hasReceipt(transaction: Transaction | undefined | null): boolean {
 }
 
 function areRequiredFieldsEmpty(transaction: Transaction): boolean {
```

Comment:
cc @luacmartins  @mountiny can you guys check this change? `areRequiredFieldsEmpty` was returning false even tho we had a missing modifiedMerchant, because it doesn’t check for non existent modifiedMerchant in the object